index.html

```diff
@@ -1 +1,38 @@
-My Page
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Facebook Clone</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" type="text/css" media="screen" href="homepage.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="header">
+        <div class="nav-bar">
+          <h1 class="logo">facebook</h1>
+          <div class="login">
+            <ul class="login-input">
+              <li class="list">
+                <label for="email" class="label">Email or phone</label>
+                <input type="text" />
+              </li>
+              <li class="list">
+                <label for="password" class="label">Password</label>
+                <input type="password" />
+                <a href="" class="forgot-acct">Forgot account?</a>
+              </li>
+
+              <li class="list submit-button">
+                <input value="Log In" class="sbmt" />
```

Comment:
Rename this class `sbmt` to `login-btn` because sbmt is not intuitive enough for another developer reading your code